src/composables/secureStorageRef.ts

```diff
@@ -8,13 +9,10 @@ import { type ICreateStorageRefOptions, useStorageRef } from './storageRef';
 interface ICreateSecureStorageRefOptions<T> extends
     Omit<ICreateStorageRefOptions<T>, 'serializer' | 'enableSecureStorage' | 'onBackgroundSync'> {
   encryptionDisabled?: boolean;
+  isStoredUnencrypted?: (val: T | null) => boolean;
```

Comment:
I still prefer to have separate encrypted and not encrypted values and compare them whenever I want in the code to establish things.